lib/views/pages/login_signup/register_form.dart

```diff
@@ -70,7 +70,7 @@ class RegisterFormState extends State<RegisterForm> {
       setState(() {
         _progressBarState = false;
       });
-      _exceptionToast(result.exception.toString().substring(16));
+      _exceptionToast('Sign Up unsuccessful. Please try again');
```

Comment:
Please change the message to "Invalid organization URL". It will give user clear idea what is going wrong.

Reply:
Sure! thing!